FAQ.md

```diff
@@ -23,7 +23,7 @@ this.setState({ editorState: EditorState.createEmpty() })
 
 ## Why are mentions broken after using `convertFromRaw` and throwing an error?
 
-We design the API to accept an Immutable Map for a Mention. After saving your datastructure to the erver and using `convertFromRaw` the mentions in there are plain objects. I (Nik) believe this is a law in our design and the mention data should be a plain object. Hint: We might fix this with v2.0.0 f the mentions plugin.
+We design the API to accept an Immutable Map for a Mention. After saving your data structure to the server and using `convertFromRaw`, the mentions in there are plain objects. I (Nik) believe this is a law in our design and the mention data should be a plain object. Hint: we might fix this with v2.0.0 of the mentions plugin.
```

Comment:
Good points, thanks for sharing 🍻 

>  callers shouldn't worry about what goes on under the hood

Agreed but right now there are other places where you do need to use Immutable objects, e.g the suggestions. Maybe we should change those too.

> DraftJS works when using convertFromRaw so I imagine they're already doing the conversion

If DraftJS does it then I think it's safe to follow their example. All in all I'd personally be ok with allowing JavaScript and Immutable objects for the public api.